src/components/RevievCard.jsx

```diff
@@ -0,0 +1,45 @@
+import { useState, useEffect } from "react"
+import { useParams } from "react-router-dom"
+
+
+function ReviewCard () {
+
+const {id} = useParams()
+const [revCard, setRevCard] = useState(null)
+const [isLoading, setIsLoading] = useState(true);
+
+useEffect(() => {
+    fetch(`https://db-reviews.onrender.com/api/reviews/${id}`)
```

Comment:
The calls to the api should to be extracted into a separate module. for instance: `./src/utils/api.js` and controller functions exported from there with semantic names like `fetchReviewByID( )` or `getIndividualReview( )`